wiki/advantages-and-features.md

```diff
@@ -0,0 +1,185 @@
+---
+title: Gridcoin Advantages & Features
+layout: wiki
+---
+
+
+# Competitive Advantages 
+
+### General Advantages
+
+One of the main advantages to Gridcoin is that crunching ("mining")
+does useful scientific research. Crunching helps science in areas like fighting 
+diseases, finding asteroids, understanding subatomic particles, and more. In other
+cryptocurrencies, mining doesn't produce any meaningful results. This means that 
+regardless of the number of Gridcoin you earn, crunching alone achieve something 
```

Comment:
```suggestion
regardless of the number of Gridcoin you earn, crunching alone achieves something 
```

Reply:
Oops not sure how I missed this one. Fixed that